proposed/form.md

```diff
@@ -0,0 +1,101 @@
+# Parsy Pattern Design Doc
+
+Parsy
+
+#Form
+
+* Author: `Lorenzo Wallerlei, lorenzo@polderknowledge.nl`
+* Date: `19-01-2017`
+* Repository name: `Form`
+
+## Goals
+
+This document is the proposal for the form component, it describes what should go in a form and how a form should react 
+to user input.
+
+## References
+
+http://getbootstrap.com/components/#input-groups
+
+http://foundation.zurb.com/sites/docs/forms.html
+
+http://webaim.org/techniques/forms/controls
+
+## Requirements
+
+- All the inputs within the form should contain a tab index.
```

Comment:
Words like MUST,SHOULD,COULD - MUST be in capitals. to make clear that these are part of the MoSCoW list